agent/acs/session/payload_responder.go

```diff
@@ -132,6 +132,12 @@ func (pmHandler *payloadMessageHandler) addPayloadTasks(payload *ecsacs.PayloadM
 			loggerfield.DesiredStatus: apiTask.GetDesiredStatus(),
 		})
 
+		if apiTask.EnableFaultInjection {
```

Comment:
nit: Could we use `IsFaultInjectionEnabled()` instead?

Reply:
We could use `IsFaultInjectionEnabled()` instead, but used `apiTask.EnableFaultInjection` to maintain similarity in the `addPayloadTasks` with similar such checks.

Reply:
Er sorry, I meant could we use the following getter function -> https://github.com/aws/amazon-ecs-agent/blob/08341f7b92d1991feac4eb246c87063d19375f9d/agent/api/task/task.go#L3770

Reply:
Also btw, the other checks are referring to another task object(ecsacs task) and not the `apitask`

Reply:
Changed in [74c3067](https://github.com/aws/amazon-ecs-agent/commit/74c3067b008ffcddd4db3b61e39d2f7238cecacd)